app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -870,24 +870,6 @@ abstract class BaseBrowserFragment :
         context.settings().incrementSecureWarningCount()
     }
 
-    @VisibleForTesting
-    internal fun expandToolbarOnNavigation(store: BrowserStore) {
```

Comment:
Would it be possible to just 
- rename this method to something like `closeFindInPageBarOnNavigation()` or `updateUIOnNavigation()` (if wanted to give it a more generic name) and 
- remove the `browserToolbarView.expand()` call?

Reply:
I'll update as you've mentioned.

Reply:
@Mugurell, I've committed the changes